dbt/include/redshift/macros/adapters.sql

```diff
@@ -51,6 +53,22 @@
   as (
     {{ sql }}
   );
+  {% else%}
+  create {% if temporary -%}temporary{%- endif %} table
```

Comment:
Can you reply with the fully compiled DDL for this? It'll be easier to review this PR.

Reply:
```sql 
BEGIN;
  create  table
    "ci"."dbt_dconnors"."test_table__dbt_tmp"
    
  (
        id integer not null ,
        color varchar ,
        date_day date ,
        primary key(id) 
    )

    
    
    
  ;

  insert into "ci"."dbt_dconnors"."test_table__dbt_tmp"
    (
      select 1 as id, 'red' as color, cast('2022-05-05' as date) as date_day
union all 
select 2 as id, 'blue' as color, cast('2022-05-05' as date) as date_day
union all 
select 3 as id, 'green' as color, cast('2022-05-05' as date) as date_day
    )
  ;
  COMMIT;
```

Reply:
this is after incorporating your above suggestions!

Reply:
Thank you!